src/templates/static.tsx

```diff
@@ -52,69 +48,150 @@ import QuizWidget from "../components/Quiz/QuizWidget"
+  return <Box my={16} w="10%" h={1} bgColor="homeDivider" />
+}
 
-  .featured {
-    padding-left: 1rem;
-    margin-left: -1rem;
-    border-left: 1px dotted ${(props) => props.theme.colors.primary};
-  }
+const Header1 = ({ children }: ChildOnlyProp) => {
+  return (
+    <Heading
+      as="h1"
+      fontSize={{ base: "2.5rem", md: "5xl" }}
+      lineHeight={1.4}
+      fontWeight="bold"
+      _before={{
+        content: `""`,
+        display: "block",
+        h: "140px",
+        mt: "-140px",
+        visibility: "hidden",
+      }}
+      sx={{
+        a: {
+          display: "none",
+        },
+      }}
+    >
+      {children}
+    </Heading>
+  )
+}
 
-  .citation {
-    p {
-      color: ${(props) => props.theme.colors.text200};
-    }
-  }
-`
+const Header2 = ({ children }: ChildOnlyProp) => {
+  return (
+    <Heading
+      fontSize={{ base: "2xl", md: "2rem" }}
+      lineHeight={1.4}
+      fontWeight="bold"
+      sx={{ position: "inherit !important" }}
+      _before={{
+        content: `""`,
+        display: "block",
+        h: "120px",
+        mt: "-120px",
+        visibility: "hidden",
+      }}
+    >
+      {children}
+    </Heading>
+  )
+}
 
-const LastUpdated = styled.p`
-  color: ${(props) => props.theme.colors.text200};
-`
+const Header3 = ({ children }: ChildOnlyProp) => {
+  return (
+    <Heading
+      as="h3"
+      fontSize={{ base: "xl", md: "2xl" }}
+      lineHeight={1.4}
+      sx={{ position: "inherit !important" }}
+      _before={{
+        content: `""`,
+        display: "block",
+        h: "120px",
+        mt: "-120px",
+        visibility: "hidden",
+      }}
+    >
+      {children}
+    </Heading>
+  )
+}
 
-const Pre = styled.pre`
-  max-width: 100%;
-  overflow-x: scroll;
-  background-color: ${(props) => props.theme.colors.preBackground};
-  border-radius: 0.25rem;
-  padding: 1rem;
-  border: 1px solid ${(props) => props.theme.colors.preBorder};
-  white-space: pre-wrap;
-`
+const Header4 = ({ children }: ChildOnlyProp) => {
+  return (
+    <Heading
+      as="h4"
+      fontSize={{ base: "md", md: "xl" }}
+      lineHeight={1.4}
+      fontWeight="semibold"
+      sx={{ position: "unset !important" }}
+      _before={{
+        content: `""`,
+        display: "block",
+        h: "120px",
+        mt: "-120px",
+        visibility: "hidden",
+      }}
+    >
+      {children}
+    </Heading>
+  )
+}
 
-const MobileTableOfContents = styled(TableOfContents)`
-  position: relative;
-  z-index: 2;
-`
+const Paragraph = ({ children }: ChildOnlyProp) => {
+  return (
+    <Text fontSize="md" mt={8} mb={4} color="text300">
+      {children}
+    </Text>
+  )
+}
 
-const HR = styled.hr`
-  width: 100%;
-  margin: 2rem 0rem;
-  margin-bottom: 1rem;
-  display: inline-block;
-  position: inherit;
-  background: ${(props) => props.theme.colors.border};
-`
+const ListItem = ({ children }: ChildOnlyProp) => {
+  return <ChakraListItem color="text300">{children}</ChakraListItem>
+}
```

Comment:
Unfortunately, this will fail a build without a declared `ul` component.

The Chakra `ListItem` component is dependent on a Chakra `List` component (either `List`, `UnorderedList`, or `OrderedList`) as a Context provider is involved for the default styling.

You will need to add Chakra's `UnorderedList` component to the components object. (And add `marginStart` value of `1.45rem` per current global styling.
```tsx

const UnorderedList = (props: ChildOnlyProp) => (
  <ChakraUnorderedList ms="1.45rem" {...props} />
)

// .... //

const components = {
  // ... //,
  ul: UnorderedList,
  li: ListItem,
  // ... //
}